### DISCUSSION TITLE
Argo multi-step workflow doesn't terminate or proceed to next step

### DISCUSSION THREAD

**Discussion by ongchinkai** [argoproj/argo-workflows#6063] [Q&A]
My actual issue can be found at https://github.com/hyfen-nl/PIVT/issues/106

TLDR While trying to use an Argo workflow to create channels on a Hyperledger Fabric network, Argo doesn't proceed to the next step in the workflow because it seems like the pod exit is not detected. Running the hello-world example at https://argoproj.github.io/argo-workflows/examples/#steps shows that the workflow is not terminating.

**Comment by kjell-ohlsson** [comment]
Hi!
I’m experiencing something very similar. See https://github.com/argoproj/argo-workflows/discussions/6131#discussion-3407968
I posted the same question in slack too and one suggestion I’ve got is to try and change executor from the default docker one. See here;
 https://cloud-native.slack.com/archives/C01QW9QSSSK/p1623418687171500
 Haven’t tried yet.
